app/javascript/v3/views/login/Index.vue

```diff
@@ -193,9 +200,46 @@ export default {
           if (this.email) {
             window.location = '/app/login';
           }
+          this.loginApi.hasErrored = true;
           this.showAlert(response?.message || this.$t('LOGIN.API.UNAUTH'));
         });
     },
   },
 };
 </script>
+
+<style scoped>
+.login--error {
```

Comment:
@pranavrajs let me know what you think of this animation

Reply:
Animation looks fine. Let us not write classes like this. Can we rewrite this in tailwind helpers?